modules/core/src/main/java/org/apache/ignite/internal/sql/command/SqlAlterUserCommand.java

```diff
@@ -36,6 +36,22 @@
     /** User's password. */
     private String passwd;
```

Comment:
Let's annotate the field with @IgniteToStringExclude

Reply:
Thanks, fixed